pkg/common/aws/utils.go

```diff
@@ -18,34 +19,36 @@ import (
 type ccsAwsSession struct {
 	session *session.Session
 	iam     *iam.IAM
+	ec2     *ec2.EC2
 	once    sync.Once
 }
 
 // CCSAWSSession is the global AWS session for interacting with AWS.
 var CcsAwsSession ccsAwsSession
 var ccsKeys *iam.CreateAccessKeyOutput
 
-//GetSession returns a new AWS session with the first AWS account in the config file. The session is cached for the rest of the program.
-func (CcsAwsSession *ccsAwsSession) getIamClient() (*session.Session, *iam.IAM) {
+// GetSession returns a new AWS session with the first AWS account in the config file. The session is cached for the rest of the program.
+func (CcsAwsSession *ccsAwsSession) getClient() (*session.Session, *iam.IAM, *ec2.EC2) {
 	var err error
 
 	CcsAwsSession.once.Do(func() {
 		CcsAwsSession.session, err = session.NewSession(aws.NewConfig().
-			WithCredentials(credentials.NewStaticCredentials(viper.GetString("ocm.aws.accessKey"), viper.GetString("ocm.aws.secretKey"), "")).
+			WithCredentials(credentials.NewStaticCredentials(viper.GetString(config.AWSAccessKey), viper.GetString(config.AWSSecretAccessKey), "")).
 			WithRegion(viper.GetString(config.CloudProvider.Region)))
 		CcsAwsSession.iam = iam.New(CcsAwsSession.session)
+		CcsAwsSession.ec2 = ec2.New(CcsAwsSession.session)
 	})
 	if err != nil {
 		log.Printf("error initializing AWS session: %v", err)
 	}
 
-	return CcsAwsSession.session, CcsAwsSession.iam
+	return CcsAwsSession.session, CcsAwsSession.iam, CcsAwsSession.ec2
 }
 
 // AWS check for osdCCSAdmin credentials
 func VerifyCCS() (string, error) {
 	var err error
-	CcsAwsSession.session, CcsAwsSession.iam = CcsAwsSession.getIamClient()
+	CcsAwsSession.session, CcsAwsSession.iam, CcsAwsSession.ec2 = CcsAwsSession.getClient()
```

Comment:
```suggestion
	CcsAwsSession.session, CcsAwsSession.iam, _ = CcsAwsSession.getClient()
```

I don't understand why we are setting these right back to `CcsAwsSession.*` when that is where `getClient` returns them from

Reply:
Pushed new PR. 
Removed the returns as well since the CcsAwsSession can be shared across packages.